Release-Checklist.md

```diff
@@ -0,0 +1,79 @@
+Change release process, instead create a release candidate we can test the HEAD what will be the last commit using new automated obs package generation we can generate and test commits based on HEAD.
+
+- [ ] Raise PRs to update the `VERSION` file in the following repositories:
+   - [ ] [agent][agent].
+   - [ ] [proxy][proxy].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+  Note that the "phase" element of the project encoded in the version strings needs to match for all components. For example, when a `beta` is released, the version string for *all* components should show `beta`.
+
+- [ ] Ensure all CI tests pass **for all architectures**.
+
+- [ ] Get confirmation from metrics CI that performance is within acceptable limits.
+
+- [ ] Create an **annotated tag** for the new release version for the following repositories:
+   - [ ] [agent][agent].
+   - [ ] [proxy][proxy].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+   This is required by `git describe`.
+
+- [ ] Generate OBS packages based on `HEAD`:
+   - [ ] [agent][agent].
+   - [ ] [guest kernel][kernel].
+   - [ ] [image][image].
+   - [ ] [initrd][initrd].
+   - [ ] [proxy][proxy].
+   - [ ] [qemu-lite][qemu-lite].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+- [ ] Installation tests (must be done for major releases)
+  - [ ] Centos.
+  - [ ] Fedora.
+  - [ ] Ubuntu.
+
+- [ ] Check if any of the following need to be updated:
+ - [ ] Architecture document.
```

Comment:
I think you need double spaces here? - they did not indent on my github md doc view window.

Reply:
Fixed.